src/ontology/uberon-edit.obo

```diff
@@ -224033,6 +224033,17 @@ is_a: UBERON:0000029 ! lymph node
 relationship: dc-contributor http://orcid.org/0000-0002-1773-2692
 property_value: dcterms-date 2022-05-26T11:59:51Z xsd:dateTime
 
+[Term]
+id: UBERON:8470002
+name: septomarginal trabecula
+def: "A muscular trabecula in the right ventricle of the heart that links the lower part of the interventricular septum to the wall and to the base of the anterior papillary muscle of the right ventricle. The morphology and structure of the septomarginal trabecula varies in different species." [https://doi.org/10.1016/j.morpho.2010.03.004, https://doi.org/10.19080/APBIJ.2017.02.555595, https://orcid.org/0000-0001-7655-4833, PMID:30984450]
+comment: In humans, the septomarginal trabecula connects the interventricular septum with the anterior wall of the right ventricle at the base of the anterior papillary muscle. It is also present in non-human mammals where it stretches between the bases of the muscles papilaris subarteriosus and magnus. In some instances it carries the continuation of the right bundle branch of the conducting system of the heart. It is constituted of ordinary cardiac myocytes around and in between Purkinje myocytes. {xref="https://doi.org/10.1016/j.morpho.2010.03.004", xref="https://doi.org/10.19080/APBIJ.2017.02.555595"}
+synonym: "moderator band" EXACT [https://doi.org/10.1016/j.morpho.2010.03.004]
+is_a: UBERON:0000440 ! trabecula
+relationship: dc-contributor https://orcid.org/0000-0002-1773-2692
+relationship: located_in UBERON:0002080 ! heart right ventricle
```

Comment:
Relation guide would be useful here >.< I'm not 100% sure about located in and would need context that maybe @dosumis could provide. (I think it is safe but would rather get someone who knows better)
heart right ventricle seems to be a bit confusing in how its modelled with the comment on cardiac chamber:
"generic enough to cover FBbt:00003156 heart chamber but this is a cavity. GO defines it as the cavity. TODO - move subclasses. Note this also includes sinus venosus" yet it *seems* material? 
Located in seems to not want it to be a spatial region having both range and domain being
```
'independent continuant'
 and (not ('spatial region'))
```
but I guess not doesn't work with ELK so that might not have been really enforced?

Reply:
A ventricle is the hollow part of an organ (in this case the heart). That is why I chose "located in" as opposed to 'part of'.

Reply:
I agree with you, just not sure if it is modelled that way in uberon (but i might be wrong) it's not super clear to me from a brief look, but I think if you trace back, it leads to something material.

Reply:
![Screenshot 2022-12-13 at 16 19 46](https://user-images.githubusercontent.com/76212760/207387000-9c98939d-e04e-4af2-bd43-02a5c1114426.png)

Reply:
At initial glance, this appears poorly modelled.

Text definition of ['cardiac chamber'](http://purl.obolibrary.org/obo/UBERON_0004151):
"A cardiac chamber surrounds an enclosed cavity within the heart."

['cardiac ventricle'](http://purl.obolibrary.org/obo/UBERON_0002082), which should be the cavity, is modelled as a subclass of 'cardiac chamber'.
If the ventricle is surrounded by the chamber it does not seem correct to model the ventricle as a type of chamber.

Reply:
Proposal:

use 'part of' some 'cardiac chamber' 
instead of 
'located in' in some 'cardiac ventricle'

and address ventricle modelling in separate ticket.

Reply:
I think it will eventually hit the same problem conceptually, but part_of is a clear pattern and I think that would be safe. 100% agree that tricky modelling thing should be in a separate ticket and not delay this unnecessarily :)

Reply:
Replaced 'located in' axiom with 'part of' axiom.
Also updated text def to make more general given the various morphologies reported and added another reference.